### DISCUSSION TITLE
zotero7安装最新版本后没有笔记工作区

### DISCUSSION THREAD

**Discussion by 30755** [windingwind/zotero-better-notes#953] [Q&A]
在顶部的下拉菜单里能看到better notes的几个选项，但是在zotero主体部分没有笔记工作区（小黄标也没有）。

**Comment by ulahcherubim** [comment]
BZN plugin has implemented a new workflow. Workspaces are removed. Now you can link any note to another note via menu.

**Reply by npj06** [reply]
Sorry to bother you, I still don't figure out how to build a new note and insert plates into the note without workspace

**Reply by windingwind** [reply]
#995
